model/shared/src/main/scala/explore/model/Execution.scala

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import io.circe.Decoder
+import lucuma.core.model.sequence.ExecutionDigest
+import lucuma.odb.json.sequence.given
+
+case class Execution(digest: Option[ExecutionDigest])
```

Comment:
It's just wrapper around an optional ExecutionDigest? I assume this is just to make decoding easier?

Reply:
It's because an observation has a `Execution` which has a `ExecutionDigest`, but not directly. There's more in `Execution` which we don't use. So thinking about it it should probably stay here as it's explore-specific